.github/workflows/run-tests.yml

```diff
@@ -115,7 +115,7 @@ jobs:
         shell: bash
         run: |
           # We need to determine the Tarball's name first.
-          TARBALL_NAME=$(ls ~/downloads/tarball | grep salesforce-plugin-code-analyzer-5\\.0\\.0-alpha\\.[0-9]*\\.tgz)
+          TARBALL_NAME=$(ls ~/downloads/tarball | grep salesforce-plugin-code-analyzer-5\\.0\\.0-beta\\.[0-9]*\\.tgz)
```

Comment:
Verified that there are no remaining usages of "alpha" in our codebase aside from as part of the words "alphabetical" or "alpha-numeric".